index.d.ts

```diff
@@ -292,7 +295,9 @@ declare class Emittery<
 	emitter.emit('🐶', '🍖'); // Nothing happens
 	```
 	*/
-	once<Name extends keyof AllEventData>(eventName: Name): Promise<AllEventData[Name]>;
+	once<Name extends keyof AllEventData>(
+		eventName: Name
+	): Promise<AllEventData[Name]>;
```

Comment:
Don't do unrelated changes.

Reply:
Sorry, it was auto run by Prettier, I will remove the unrelated changes